core/common/src/main/java/alluxio/client/file/URIStatus.java

```diff
@@ -40,14 +40,17 @@
 @ThreadSafe
 public class URIStatus {
   private final FileInfo mInfo;
+  /** Context related to Presto runtime associated with this URI. */
+  private PrestoContext mPrestoContext;
```

Comment:
any reason we do not make mPrestoContext final?

Reply:
changed to `final`. thanks